tests/river/unit/transformer/test_reference_binder.py

```diff
@@ -6,8 +6,12 @@
 from river.transformer.reference_binder import ReferenceBinder
 
 
-def test_identifier_to_reference(identifier):
-    binder = ReferenceBinder()
+@pytest.fixture(scope="module")
+def binder():
+    return ReferenceBinder()
```

Comment:
nit: I don't really get why you changed this (since it costs 0 to instantiate a ReferenceBinder and it doesn't reduce code complexity) but ok

Reply:
I just didn't really like having the line repeated at the beginning of each test. And if we need to change the initialization of this binder at some point, we'll only have to change it here.
I can undo that if you prefer it the other way 🤷‍♂️